### DISCUSSION TITLE
consistency

### DISCUSSION THREAD

**Discussion by 0323pin** [getreu/tp-note#18] [General]
Hi,

Updating the NetBSD package to the 1.23.8 release, and ...
... sorry to bring this up but, would you mind keeping consistency across releases?
Sometimes it's `version number` and some other times it's `vversion number`.

This causes trouble when updating package source fetches, so sticking to one would be nice.
I've just spent 15 - 20 min changing our package `Makefile` to fetch from the correct source.
Thank you!

**Comment by 0323pin** [comment]
https://gitlab.com/getreu/tp-note/-/commit/bcaf476848619ce3377340ff7df1f11569154021
Thanks!

Will merge 1.23.9 in a few minutes.

**Comment by getreu** [comment]
You are welcome. I am sorry for the inconvinience.
 

Gesendet: Mittwoch, 28. Februar 2024 um 20:49 Uhr
Von: "pin" ***@***.***>
An: "getreu/tp-note" ***@***.***>
Cc: "Subscribed" ***@***.***>
Betreff: Re: [getreu/tp-note] consistency (Discussion #18)


 

https://gitlab.com/getreu/tp-note/-/commit/bcaf476848619ce3377340ff7df1f11569154021
Thanks!

Will merge 1.23.9 in a few minutes.
